### ISSUE TITLE
enhancement: Allow the user to choose the version returned in getVersion()

### ISSUE THREAD

**Issue by Toad06** [ruffle-rs/ruffle#819]
I don't know if it's worth the effort these days, but I remember a few old games became unusable as of Flash 10, because of a poor way to detect the Flash version:
```
ver = Number(getVersion().charAt(4));
if(ver < 7)
{
   attachMovie("ID_VERSIONERR","mcErr",100);
   stop();
}
```

This is the detection code from "Uncle Sam":
https://www.y8.com/games/uncle_sam

Ruffle could eventually relive them by allowing the user to choose the version returned in getVersion() (or maybe Ruffle could just pretend to be version 99... 😁 though this may also have downsides).

**Comment by Dinnerbone** [comment]
We already have some semblance of which version of the Flash player we're emulating, and the idea is that we can use this for the breaking changes introduced around Flash 4-6. Once we have a way to select that, this should be a natural extension.

**Comment by Herschel** [comment]
Here's an SWF with a broken version check that would benefit:
https://www.newgrounds.com/portal/view/294434
(broken version check considers Flash Player 32 to be Flash Player 3).

**Comment by n0samu** [comment]
This has been implemented in #8961. Not working for AVM2 yet though - see #9180,
